lib/extensions.js

```diff
@@ -63,6 +64,8 @@ function getBinaryName() {
     binaryName = flags['--sass-binary-name'];
   } else if (process.env.SASS_BINARY_NAME) {
     binaryName = process.env.SASS_BINARY_NAME;
+  } else if (process.env.npm_config_sass_binary_name) {
+    binaryName = process.env.npm_config_sass_binary_name;
```

Comment:
I only know `npmconf` as NPM module and not as an env namespace. I still think `npm_config_` is the right thing :smile:. As per NPM documentation:
> Any environment variables that start with npm_config_ will be interpreted as a configuration parameter.

This mechanism works bidirectional and npm exposes all configuration parameter into this env namespace which will be then passed to all child processes (like in the phantom.js installation). Parameters could be set in a local/global .npmrc file or passed directly as argument.

I also tested it locally using a .npmrc file in the project with an alternative download path and it worked like a charm.